checkstyle.json

```diff
@@ -517,6 +517,22 @@
 		},
 		{
 			"props": {
+				"severity": "INFO",
+				"ignorePackages": [
```

Comment:
I guess not being able to detect "implicit imports" (types that are used, but only imported implicitly because they are also inside the imported module) are a necessary limitation?